contrib/automation_tests/orbit_memory_tracing.py

```diff
@@ -29,9 +29,9 @@ def main(argv):
         SetAndCheckMemorySamplingPeriod(memory_sampling_period='ab'),
         SetAndCheckMemorySamplingPeriod(memory_sampling_period='0'),
         Capture(collect_system_memory_usage=True),
-        FilterTracks(filter_string="Memory", expected_count=1),
+        VerifyTracksExist(track_names="Memory*"),
         Capture(),
-        FilterTracks(filter_string="Memory", expected_count=0)
+        VerifyTracksExist(track_names="Memory*")
```

Comment:
Maybe `VerifyTracksDoNotExist` here?

Reply:
Oh, this is actually a bug, thanks for pointing that out. Will fix that.

Reply:
Fixed